main.cpp

```diff
@@ -0,0 +1,30 @@
+#include <iostream>
+
+int main() {
+    std::cout << "Enter shares bought." << std::endl;
+    int shares_bought = 0;
+    std::cin >> shares_bought;
+    std::cout << "Enter cost."  <<std::endl;
+    double purchase_cost = 0.0;
+    std::cin >> purchase_cost;
+    double COMMISSION_RATE = .02;
+    double purchase_commission = 0;
+    double ttl_commission;
```

Comment:
This variable name is not very meaningful. Also `l` and `1` look very similar so using them in an acronym is risky business